src/uploads.ts

```diff
@@ -125,7 +125,7 @@ export async function toFile(
     }
   }
 
-  return new File(bits, name, options);
+  return new File(bits as (string | Blob | NodeJS.ArrayBufferView)[], name, options);
```

Comment:
I couldn't for the life of me figure out how to get this type to work. Help appreciated!

Reply:
heh, I believe it, can try to take a look soon… likely thursday…

Reply:
@Ethan-Arrowood I think `undici`'s type for the `fileBits` parameter is wrong, their own JSDoc says it accepts `ArrayBuffer`s, but the type annotation only accepts `ArrayBufferView`s, which `ArrayBuffer`s aren't assignable to.